### ISSUE TITLE
RTV development is shutting down

### ISSUE THREAD

**Issue by michael-lazar** [michael-lazar/rtv#696]
# RTV development is shutting down

Hi, I'm the author and sole maintainer of RTV (https://github.com/michael-lazar/rtv).

This is an official announcement that I will no longer be maintaining this project.

### Why?

Lately, RTV has stopped being fun for me to work on. I suspect this is the typical burnout that many open source maintainers face. I have a limited amount of free time, and other projects that I would rather be spending it on. 

More concerning, I no longer wish to support reddit as a platform. Widespread astroturfing, manufactured controversy, politics, bullying, incessant nitpicking and arguing. 

For a while I held
out hope that this was limited to the "front page", and that smaller subreddits
were still redeemable. But lately I've observed even niche subreddits become
subjected to the above garbage. 

 I don't like how people treat each other on the platform, and I don't like how it makes me feel when I use it.

So I'm making a clean break and moving on to other things that make me happy.

### What will happen to RTV?

Nothing, the GitHub repo will remain as it is today. I am simply going to stop responding to issues and pull requests. I suspect that many people will continue to use the latest release of RTV for some time, until reddit inevitably breaks their API. I may archive the repo sometime in the future.

I will retain access to the [PyPI repo](https://pypi.org/project/rtv/) and will not publish any new releases. I don't have any involvement in the other package repositories. It's up to those maintainers to decide how to handle this.

I will not be offering any future security patches or bugfixes.

For those who want to continue development, I suggest that forking is the appropriate course of action. I support anyone who wants to fork this project, provided that you follow to these guidelines:

1. Change the name of your fork, so people don't confuse it with this project (fun fact, the name *"Reddit Terminal Viewer"* is a direct violation of reddit's [usage policy](https://www.reddit.com/wiki/api)).
2. Attribute RTV as the source and provide a link back to this project's GitHub repo.

### Why not transfer ownership to someone else?

Frankly, I don't want to put the work into vetting a new maintainer or transferring ownership to an organization. Stuff like [this](https://github.com/dominictarr/event-stream/issues/116) scares me.

Forking is nice because it allows the new maintainer to establish their own trust and reputation within the community. It's also nice because it requires zero involvement from myself. I acknowledge that this is inconvenient for the average RTV user. Change is always annoying, especially when you have no control over it.

### Thanks

I would like to say thank you to everyone who interacted positively via the issue tracker, pull requests, email, [saythanks.io](https://saythanks.io/to/michael-lazar), reddit, and other channels. I look forward to building more things together in the future!

**Comment by goggle** [comment]
Oh, that's really sad... but I can understand you. If the project doesn't satisfy you anymore, it's the right thing to do! I've known and used `rtv` for many years. It was a great project! Thank you for putting all that effort in it!

Unfortunately, I've only recently started to contribute to `rtv`, so I have some open PRs that did not make it into that project: https://github.com/michael-lazar/rtv/pull/684, https://github.com/michael-lazar/rtv/pull/688 and https://github.com/michael-lazar/rtv/pull/691. I hope that we can establish a successful fork.

**Comment by michael-lazar** [comment]
@goggle Thanks! Sorry about those PRs. It was nothing personal or a knock against your code, just bad timing.

**Comment by woorst** [comment]
@michael-lazar it's been real. Thanks for this great project

**Comment by cyberjunkie2k** [comment]
Just a user, but I am very thankful for all your work and effort. It's been an excellent project. I hate the direction popular (social and news) media has taken over the past few years, and I can relate to your frustration.

Hope you find your time better used in more productive, satisfying and gratifying things. Thanks again!

**Comment by gbpi** [comment]
Thank you for your great work. I only recently started using rtv and have been really enjoying it.

I feel like rtv makes the platform more tolerable, in a browser I would find myself endlessly scrolling /all/ and that rarely leads anywhere good. Now I just have a couple multis setup with some subs I enjoy and check in on them occasionally with rtv.

**Comment by Pac23** [comment]
Thank you for the work you did with RTV,it was a excellent terminal based reddit tool.

As far as reddit goes,the increasing no of "normal" folks moving to reddit from sites like facebook imo is the reason for its downfall.It used to be a great community/knowledge resouce especially for hobbies and whatnot sadly with "geek/nerd is the new cool" thing destroying everything in its path,reddit too seems to have fallen.

**Comment by manskron** [comment]
I've enjoyed RTV very much. Thanks for all the work you've done on it.

**Comment by naiveai** [comment]
Thank you for the work you did with RTV! It's a great thing you've made here.

**Comment by bart-kneepkens** [comment]
Your concerns for the platform are valid. Thanks for making RTV!

**Comment by bronikowski** [comment]
Can't blame you, I really liked to have a low distraction client that runs great on RPi. Have fun with your next project!

**Comment by vor0nwe** [comment]
That’s too bad, but understandable.

Are you going to [archive](https://help.github.com/en/articles/archiving-a-github-repository) this repo? Makes it clearer that it’s no longer being maintained.

**Comment by goggle** [comment]
@vor0nwe He answered that question:

> Nothing, the GitHub repo will remain as it is today. I am simply going to stop responding to issues and pull requests. I suspect that many people will continue to use the latest release of RTV for some time, until reddit inevitably breaks their API. I may archive the repo sometime in the future.

**Comment by rudevdr** [comment]
Its sad that the project is being shut down. But I am sure the source code will help many new comers learn many things about programming.

**Comment by michaelachrisco** [comment]
Thank you so much for your work over the years! After the redesign of reddit, I put rtv on almost all of my systems because of the clean interface.

**Comment by dessalines** [comment]
Thank you so much for your work, this is totally understandable. Who knows how long it'll be before reddit closes off their API and goes the way of twitter and FB. 

For anyone else interested, I've been building  a federated reddit alternative called [Lemmy](https://github.com/dessalines/lemmy) intended to work in the fediverse, and have an [open ticket](https://github.com/dessalines/lemmy/issues/152) for someone to fork RTV to get a working command line crest for it.

**Comment by skorokithakis** [comment]
Please consider adding the project to the [Code Shelter](https://www.codeshelter.co/), that way someone vetted might pick it up at some point.

**Comment by kulkarniamit** [comment]
Convenient keyboard shortcuts, intuitive distraction free interface, help for commands, ease of use, colors for titles, easiest way to toggle threads, and many more useful features. I thank you for all of this! Wishing you success in your next productive project 👍 You've set an awesome example for clean distraction free command line interfaces. Loads to learn from your work 👍

**Comment by ruifm** [comment]
Great project, such a shame that it has come to an end but it is completely understandable. Do you think in the future you could post 1 or 2 forks you deem worthy on your README?

**Comment by miguelmota** [comment]
Sad to see you leave the project and it's completely understandable; at some point it becomes a chore that is no longer enjoyable. We are grateful for RTV and wish you the best!

**Comment by Brutus11111** [comment]
Really sad to hear. I use this app on a daily basis. I do understand your reasons. Reddit isn't getting better, but it does bring me good information on specific subjects. I hope someone else can fork it and continue the great work you have done so far.

**Comment by michael-lazar** [comment]
> Great project, such a shame that it has come to an end but it is completely understandable. Do you think in the future you could post 1 or 2 forks you deem worthy on your README?

Absolutely! People are also welcome to use this project's issue tracker to promote alternative projects that they're working on.

**Comment by thy6** [comment]
I love this application as it makes reddit so much better to use. Thank you so much for making it. I can understand your reasons to no longer support it. I hope the development will continue in another fork, I would really like this application to become platform independent and support not just reddit but every other website like it: hackernews, voot, raddle, saidit, etc.

**Comment by adam-lee** [comment]
Darn, I came to check up on the status of the project after having used it last year. Hopefully someone forks and continue on.

**Comment by ajakk** [comment]
@adam-lee I'm maintaining a fork [here](https://gitlab.com/ajak/tuir)

**Comment by github-userx** [comment]
<3 thanks for everything.

**Comment by deepend-tildeclub** [comment]
If you ever want to transfer ownership to someone our little community (tilde.club) would happily take it on..  But unlikely that you'd do that as you stated, I have put it on our github: 
https://github.com/tildeclub/ttrv

Thanks for great software over the years :)

**Comment by github-userx** [comment]
To be honest, I’m glad Michael still owns the pypi/pip package and github repo.

**Comment by himanshuxd** [comment]
It was awesome trying it out.

**Comment by jeromefaria** [comment]
I just noticed this post moments after commenting on the latest issue (sorry about that!). Occasional Reddit user here but `rtv` has been my favourite way to browse it for a couple of years. Sorry to see it go. Best of luck with your future projects and thanks for all the hard work that went into this one!

**Comment by richmont** [comment]
I've reinstalled RTV now and see this
Please press F
